doc/template.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>__SECTION__ io.js __VERSION__ Manual &amp; Documentation</title>
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700,400italic">
+  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700,400italic">
```

Comment:
Protocol-relative URLs on HTTPS-enabled documents are an antipattern. Why not just always use the HTTPS?